tests/mock_custom/nrf_modem_at/Kconfig

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MOCK_CUSTOM_NRF_MODEM_AT
+	bool "Custom mock for nrf_modem_at"
```

Comment:
similar to the other comment regarding `custom mock`, should we just use `nrf_moden_at mock configuration` here ?
```
menuconfig MOCK_NRF_MODEM_AT
	bool "nrf_modem_at mock configuration"
```

We could in addition have a
```
config CMOCK_GENERATE
    bool "CMock generated mocks"
    default y
```
elsewhere in the Kconfig tree if such setting would make sense.

That would kind of make the distinguish between (hand-written) mocks and CMake generated mocks clearer.